src/js/components/Select/stories/ChildrenRender.js

```diff
@@ -85,7 +85,7 @@ export const Children = () => {
         selected={selected}
```

Comment:
@sabarnix would you be open to changing the story a bit..

When this story was made since then `selected` is actually going to be deprecated. I figured since we are in the files if you can make a few quick adjustments. 
 https://v2.grommet.io/select#selected

remove the `selected={selected}` then use `value={selected}` then with what we have in our `value` now can you move that to be 
```
        valueLabel={
          <Box wrap direction="row" width="small">
            {selected && selected.length ? (
              selected.map((index) => renderSeason(allSeasons[index]))
            ) : (
              <Box
                pad={{ vertical: 'xsmall', horizontal: 'small' }}
                margin="xsmall"
              >
                Select Season
              </Box>
            )}
          </Box>
        }
```

then update the `onChange` I believe this will work the same just more aligned to what our current guidance is.

Reply:
done